apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -204,25 +207,6 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
               as="p"
             />
           </div>
-          {!booking && (
```

Comment:
There is no case in this component where booking isn't set. Params types also updated to reflect that.